code/player/WalkController.cs

```diff
@@ -4,11 +4,35 @@
 
 namespace TTT;
 
+
+public class DuckSpeed : Duck
+{
+	// The default Duck speed is 64 as of writing.
+	// Due to what I asume is friction, that makes you move like a SNAIL.
+	// It's also hardcoded so it requires a custom class override.
+
+	public float Speed = 80.0f;
+
+	public DuckSpeed( BasePlayerController controller, float fSpeed ) : base( controller )
+	{
+		Speed = fSpeed;
+	}
+
+	public override float GetWishSpeed()
+	{
+		if ( !IsActive ) return -1;
+		return Speed;
+	}
```

Comment:
I think this class should literally just be an override on `GetWishSpeed` with a proper `return X` where X is the appropriate speed as compared in source.